lux/vis/Vis.py

```diff
@@ -351,6 +351,17 @@ def refresh_source(self, ldf):  # -> Vis:
             self._source = ldf
             self._inferred_intent = Parser.parse(self._intent)
             Validator.validate_intent(self._inferred_intent, ldf)
+            vlist = [Compiler.compile_vis(ldf, self)]
```

Comment:
This seems redundant with the `compile_vis` and `execute` that happens subsequently. Is there a reason why we are doing this twice?